charts/keycloak-operator/templates/cluster_role.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I'd like to see some comments in this about what permissions are needed for the operator in english. This will help us understand WHY all the permissions selected here are reasonable.